contracts/FjordDrop.sol

```diff
@@ -175,9 +174,9 @@ contract FjordDrop is Erc721BurningErc20OnMint, ReentrancyGuard, IERC2981 {
         }
     }
 
-    
     function publicMint(uint256 _amount) public payable {
-    if (mintCounter == TOTAL_SUPPLY) {
+        //TODO what if mintCounter is 500 and _amount is 1000?
```

Comment:
^

Reply:
agreed. so check should be if  `mintCounter + amount > TOTAL_SUPPLY` we revert